Recipes/InheritedTypeJsonConverter/InheritedTypeJsonConverter.cs

```diff
@@ -33,11 +33,70 @@ namespace Spritely.Recipes
     [System.CodeDom.Compiler.GeneratedCode("Spritely.Recipes", "See package version number")]
 #pragma warning disable 0436
 #endif
-    internal partial class InheritedTypeJsonConverter : JsonConverter
+    internal abstract class InheritedTypeJsonConverter : JsonConverter
     {
         private readonly ConcurrentDictionary<Type, IReadOnlyCollection<Type>> allChildTypes =
             new ConcurrentDictionary<Type, IReadOnlyCollection<Type>>();
 
+        protected IEnumerable<Type> GetChildTypes(Type type)
```

Comment:
I assume the next commit or two will bring back usage of this method because I don't see any usage after this change